tools/IceRpc.Protobuf.Tools/README.md

```diff
@@ -1,2 +1,79 @@
+# Protobuf Tools for IceRPC
 
-TODO
+IceRpc.Protobuf.Tools allows you to compile Protobuf definitions (in `.proto` files) into C# code (in `.cs` files)
+within MSBuild projects.
+
+This package includes the Protobuf compiler, `protoc`, and the `protoc-gen-icerpc-csharp` generator. The `protoc`
+compiler is a native tool with binaries for Linux x64, macOS x64 and Windows (x64). The `protoc-gen-icerpc-csharp`
+generator is a .NET program and requires .NET 8 or later.
+
+Once you've added the IceRpc.Protobuf.Tools NuGet package to your project, the Protobuf files of your project are
+automatically compiled into C# files every time you build this project.
+
+[Source code][source] | [Package][package]
+
+## Protobuf files and Protobuf directories
+
+The Protobuf compiler accepts two main inputs:
+
+- the Protobuf files to compile into C# code (the Protobuf files)
+- directories that contain reference Protobuf files (the Protobuf directories)
+
+You select which files to include in your project's Protobuf files with the `ProtoFile` item type. And you select which
+directory to include in your project's Protobuf directories with the `ProtoDirectory` item type.
+
+By default, all `.proto` files located in your project's home directory and any of its subdirectories, recursively, are
+included in `ProtoFile`. You can prevent this auto-inclusion of `.proto` files by setting either
+[`EnableDefaultItems`][default-items] or `EnableDefaultProtoFileItems` to `false`. The default value of these properties
+is `true`.
+
+You can also add Protobuf files to your project explicitly. For example:
+
+```xml
+<ItemGroup>
+    <ProtoFile Include="../Greeter.proto"/>
```

Comment:
We must use the standard naming conventions for .proto files. So greeter.proto here.